packages/hub/routes/gas-station.ts

```diff
@@ -0,0 +1,32 @@
+import Koa from 'koa';
+import autoBind from 'auto-bind';
+import { inject } from '@cardstack/di';
+
+export default class GasStationRoute {
+  gasStationService = inject('gas-station-service');
+
+  constructor() {
+    autoBind(this);
+  }
+
+  async get(ctx: Koa.Context) {
+    let chainId: number = ctx.params.chainId;
+    let gasPrice = await this.gasStationService.getGasPriceByChainId(chainId);
+    if (!gasPrice) {
+      ctx.status = 404;
```

Comment:
In case there is no gas price, it means the chain is not supported right? I think it would be nice to add a message explaining this in case someone specified a chain that we don't support here

Reply:
updated